stomp/__main__.py

```diff
@@ -530,7 +537,7 @@ def main():
     heartbeats = tuple(map(int, arguments['--heartbeats'].split(",")))
 
     st = StompCLI(arguments['--host'], arguments['--port'], arguments['--user'], arguments['--password'], arguments['--protocol'],
-        prompt, arguments['--verbose'], arguments['--ssl'], heartbeats)
+        prompt, arguments['--verbose'], heartbeats, arguments['--ssl'], arguments['--ssl-key-file'], arguments['--ssl-cert-file'], arguments['--ssl-ca-file'])
```

Comment:
I think you need to reference the variable names, not the argument name (i.e. `arguments['<ssl-key-file>']`)

Reply:
thanks jason, will send new patch.